Cargo.lock

```diff
@@ -157,9 +157,9 @@ dependencies = [
 
 [[package]]
 name = "anstyle"
-version = "1.0.7"
+version = "1.0.10"
```

Comment:
Strange that there are version bumps when only a simple amend to the workspace manifest. Was this intentional?

Reply:
Not intentional . Tried to reset the cargo.lock to main and do a rebuild, but there are still version bumps. Likely due to the pop-api/integration-tests now using later versions. https://github.com/r0gue-io/pop-node/pull/434/commits/4fdfe782a71b3d7ae66e024db26d974555dba0c2